examples/grpc/tracer_common.h

```diff
@@ -38,7 +38,7 @@ class GrpcClientCarrier : public opentelemetry::context::propagation::TextMapCar
                    opentelemetry::nostd::string_view value) noexcept override
   {
     std::cout << " Client ::: Adding " << key << " " << value << "\n";
-    context_->AddMetadata(key.data(), value.data());
+    context_->AddMetadata(std::string(key), std::string(value));
```

Comment:
I think it's better to use `static_cast<std::string>(key), static_cast<std::string>(value)` here. Not all implementations of `string_view` can be used as constructor parameter of `std::string`, but most of them implement `operator std::string()`.

Reply:
For user-defined types the two are identical :)

https://godbolt.org/z/zfsz6baGr